### PR TITLE
Fix #1, cleanup code, fix eclipse and fix YUNoMakeGoodMap dependancy.

### LINKED ISSUE #1: Scanner Does Not Always Create 16x16 chunk
When placing the scanner next to another created chunk, this occurs: https://sx.artdude.uk/hotlink-ok/darkosto/CzyH-50423-1473740142.mp4

When creating a brand new chunk in the void, it generates 16x16. This only happens when next to another chunk generation.

### PR DESCRIPTION
**Pull request by LexManos**

### CODE REVIEW

File: src/main/java/eladkay/scanner/terrain/TileEntityTerrainScanner.java [hunk 1]
```diff
@@ -75,72 +78,74 @@ public void update() {
...
-                int i = ThreadLocalRandom.current().nextInt(45);
-                if (i == 0)
-                    worldObj.setBlockState(new BlockPos(x, y, z), Blocks.GOLD_ORE.getDefaultState(), 2);
-            }
         }
-        Oregistry.getEntryList().stream().filter(entry -> y < entry.maxY && y > entry.minY).forEach(entry -> {
-            int i = ThreadLocalRandom.current().nextInt(entry.rarity);
-            if (i == 0) worldObj.setBlockState(new BlockPos(x, y, z), entry.ore, 2);
-        });
+        else if (current.getY() > 64)
```
Comment (LexManos) [inline src/main/java/eladkay/scanner/terrain/TileEntityTerrainScanner.java:135]: Side note, you had this in there, I assume for debug or whatever, so I left it in. However you 'on' flag does nothing. Up above you always set it to true before checking it so ya.. its useless...

**Comment by LexManos** [comment]
You also might want to re-think your ore generation. This is what I get when I have 5 things overlapping eachother:
http://puu.sh/ra8Rm/9dbe4bd777.jpg

Without your extra oregen vanilla ores DO generator, you just need to make sure that in the other dimension you load all the chunks around where you're working as well. Should be simple enough and you only need to do it once.

**Comment by Eladkay** [comment]
The dimension is loaded. The thing is, if chunks aren't generated already at the time I access them, they don't spawn with ores. Do you have any idea how to force a chunk to generate fully? As if a player accessed it?

File: src/main/java/eladkay/scanner/terrain/TileEntityTerrainScanner.java (hunk 1 above)
Comment (Eladkay) [reply src/main/java/eladkay/scanner/terrain/TileEntityTerrainScanner.java:135]: It's used for a BlockState, it was kind of broken, sort of, but it did work and it was used

**Pull request merged**